docs/reference/installation/_interfaces/helm3.cue

```diff
@@ -0,0 +1,145 @@
+package metadata
+
+installation: _interfaces: "helm3": {
+	title:       "Helm 3"
+	description: """
+		[Helm](\(urls.helm)) is a package manager for Kubernetes that
+		facilitates the deployment and management of services on Kubernetes
+		clusters.
+		"""
+
+	archs: ["x86_64", "ARM64"]
+
+	paths: {
+		bin:         "/usr/bin/vector"
+		bin_in_path: true
+		config:      "configs/vector.{config_format}"
+	}
+
+	package_manager_name: installation.package_managers.helm.name
+	platform_name:        installation.platforms.kubernetes.name
+
+	roles: [Name=string]: {
+		commands: {
+			_name:          string
+			_resource_type: string
+			add_repo:       #"helm repo add timberio https://packages.timber.io/helm/latest"#
+			configure:      #"""
+						cat <<-VECTORCFG > \#(paths.config)
+						{config}
+						VECTORCFG
+						"""#
+			install:        #"helm install \#(_name) timberio/\#(_name) --devel --values values.yaml --namespace vector --create-namespace"#
+			logs:           #"kubectl logs -n vector \#(_resource_type)/\#(_name)"#
+			reconfigure:    #"kubectl edit \#(_resource_type) \#(_name)"#
```

Comment:
```suggestion
			reconfigure:    #"helm upgrade \#(_name) timberio/\#(_name) --devel --values values.yaml --namespace vector && kubectl rollout restart \#(_resource_type)/\#(_name)"#
```

Reply:
Why would we upgrade to reconfigure Vector?

Reply:
Upgrade in Helm replaces one release with the other. To apply different settings with the same chart version one would need to use the upgrade command. However, the version has to be fixed to avoid switching the chart version - this is something I missed in the suggestion above.